src/components/organisms/Badges/Badges.tsx

```diff
@@ -146,28 +146,28 @@ export const Badges: React.FC<{
   }
 
   return (
-    <div className="badges-component">
-      <div className="visits">
-        <div className="visit-item">
-          <span className="visit-item__value">
+    <div className="Badges">
+      <div className="Badges__visits visits">
+        <div className="visits__item">
+          <span className="visits__item-value">
```

Comment:
Why is it not aligned with our patterns?

I'd expect it to be smth like `Badges__visits`, `Badges__visit`, `Badges__visit-value`, etc

Reply:
renamed the class names: https://github.com/sparkletown/sparkle/pull/1499/commits/4b9c4fee3c3c8ac3d75c4a18814995447bfa29da